ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -314,9 +323,51 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils'],
     getInitialState: function() {
       return {
         vocabularies: [],
-        learningResourceTypes: []
+        learningResourceTypes: [],
+        message: ''
       };
     },
+    deleteVocabulary: function(vocab, dialogId) {
+      if (this.isMounted() && vocab && dialogId) {
```

Comment:
Why is `isMounted` necessary here? If `deleteVocabulary` is being called by an click handler the component should still be mounted, it's only after an AJAX call where this might not be true anymore (AFAIK).

We should not check for true-ness in `vocab` and `dialogId` because they should always be true-ish values.